temporalcli/commands.taskqueue.go

```diff
@@ -11,31 +11,237 @@ import (
 	enumspb "go.temporal.io/api/enums/v1"
 	"go.temporal.io/api/taskqueue/v1"
 	"go.temporal.io/api/workflowservice/v1"
+	"go.temporal.io/sdk/client"
 	"go.temporal.io/server/common/tqid"
 )
 
+const unversioned = "UNVERSIONED"
```

Comment:
This package is shared by many commands. May want to make the top-level constructs clearly specific to task queue work. For some of the functions this can maybe be easier putting them as methods on the command instead of top-level functions, but up to you.

Reply:
good point